content/articles/facilitate-media-queries-using-scss-mixins/index.md

```diff
@@ -0,0 +1,154 @@
+## THE BEST WAY TO FACILITATE CSS MEDIA QUERIES USING SCSS MIXINS
+### Introduction
+In web development, it is essential to build sites and applications that are user-friendly and visually appealing across every device. CSS media queries are the most used technique in creating the desired responsive websites for several screen sizes ranging from desktop to mobile. 
+
+
+### Table of contents
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [Key-takeaways](#key-takeaways)
+- [CSS Media Queries](#css-media-queries)
+- [CSS Preprocessor scripting language (SCSS)](#css-preprocessor-scripting-language-(SCSS))
+- [Creating the SCSS File](#creating-the-sccs-file)
+- [Media Queries Using SCSS Mixins](#media-queries-using-scss-mixins)
+- [Conclusion](#conclusion)
+- [Further reading](#further-reading)
+
+### Prerequisites
+To comprehend this article better, the reader should have an understanding of the following:
+- HTML and CSS 
+- Elementary knowledge of the CSS preprocessor - [SASS/SCSS](http://sass-lang.com)
+
+### Key takeaways
+By the end of this article, the reader is expected to acquire the following knowledge: 
+- How the CSS preprocessor scripting language works.
+- The best approach for handling media queries.
+- How to design responsive applications using SCSS mixins.
+- How to better implement the SCSS mixins for cleaner and easier to maintain code.
+
+### CSS media queries
+CSS media queries are among the fundamental CSS techniques that developers use to ensure the layout of a website adapts flawlessly on all devices. However, most developers agree that CSS media queries can be wordy, complex, time-consuming, and challenging to maintain. This is where the preprocessor scripting language SCSS comes in.
+
+With the aid of the CSS preprocessor, particularly SCSS, one can build tidier and easy-to-maintain websites. The preprocessor enlarges the existing features and functionalities of CSS to produce a website with responsive design and layout. The following steps cover how to create SCSS mixins. 
+
+### Step 1
+#### CSS Preprocessor scripting language (SCSS)
```

Comment:
This first step is more of a general description of CSS preprocessors. Please move it to a section before the manual steps and give it a descriptive heading. eg. `What are CSS preprocessors?`

Reply:
@geoffrey45 Please this section also includes the distinction between the preprocessors to be used, and then the preliminary stage of installing the compiler. This gives way to the next step of setting the scss file. Changing it would change the sequential order of the article. Should I still proceed? Thanks.